R/antigen.garnish_predict.R

```diff
@@ -1600,7 +1600,9 @@ up <- lapply(dtl, function(x){x[[2]]}) %>% unlist
      if ("blast_uuid" %chin% names(dt))
       dt[!is.na(BLAST_A), min_DAI := BLAST_A]
 
-    if ("DAI" %chin% names(dt)) dt[!is.na(DAI), min_DAI := min(DAI, min_DAI, na.rm = TRUE)]
+    if ("DAI" %chin% names(dt)) dt[!is.na(DAI),
+                                  min_DAI := min(DAI, min_DAI, na.rm = TRUE),
+                                  by = 1:nrow(dt[!is.na(DAI)])]
```

Comment:
yea per by other comment it is not clear to me why this did not work -- nothing wrong with extracting the v, just want to understand what is happening here

Reply:
See response to other comment. due to edge case of nrow = 0 making the by vector length 2 and > nrow(dt) (== 0)

Reply:
ah ok, got it, thanks